carp.common/src/commonMain/kotlin/dk/cachet/carp/common/data/ECG.kt

```diff
@@ -0,0 +1,8 @@
+package dk.cachet.carp.common.data
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+@SerialName( ECG_TYPE_NAME )
+data class ECG( val voltage: Double ): Data
```

Comment:
At a quick glance of what unit is commonly used, it might be more useful to store this as mV?

Reply:
I thought of that, but was unsure whether it was better to use the base SI unit in general. Some devices may report this in uV or whatnot. A double should be able to easily retain the necessary precision no matter what. However, I'm inclined to agree with you if it's a near-standard that it's always reported in mV.